ssl/s3_lib.c

```diff
@@ -842,8 +842,71 @@ static SSL_CIPHER ssl3_ciphers[] = {
      SSL_AES128GCM,
      SSL_AEAD,
      TLS1_3_VERSION, TLS1_3_VERSION,
+     SSL_kANY,
+     SSL_aANY,
+     SSL_HIGH,
+     SSL_HANDSHAKE_MAC_SHA256 | TLS1_PRF_SHA256,
```

Comment:
Well this is presumably (haven't checked the code) used to determine which algorithm to use for the HKDF: maybe change it to TLS13_HKDF_SHA256 etc?